sdk/attestation/azure-security-attestation/README.md

```diff
@@ -33,8 +33,8 @@ pip install --pre azure-security-attestation
 
 ### Authenticate the client
 
-In order to interact with the Microsoft Azure Attestation service, you'll need to create an instance of the [Attestation Client][attestation_client] or [Attestation Administration Client][attestation_admin_client] class. You need a **attestation instance url**, which you may see as "DNS Name" in the portal,
- and **client secret credentials (client id, client secret, tenant id)** to instantiate a client object.
+In order to interact with the Microsoft Azure Attestation service, you'll need to create an instance of the [Attestation Client][attestation_client] or [Attestation Administration Client][attestation_admin_client] class. You need an **attestation endpoint**, which you may see as "DNS Name" in the portal,
+and **client secret credentials (client id, client secret, tenant id)** to instantiate a client object.
```

Comment:
Can you put a link to azure-identity describing where to find information about authenticating with az-identity

Reply:
The next paragraph has links to  https://docs.microsoft.com/python/api/overview/azure/identity-readme?view=azure-python-preview. Do you have a better link suggestion?